### DISCUSSION TITLE
Components: Object Detector, Motion Detector etc.

### DISCUSSION THREAD

**Discussion by Dvalin21** [roflcoopter/viseron#556] [Ideas]
For CompreFace I know that you can do the following

cameras:
      camera_1:
      camera_2:
      camera_3:
    labels:
      - person

I would like to be able to do the same thing the other components for cameras that are all the same and if addition cameras that are added that are different, then set those individually to override the global set for the others. Thanks.

**Comment by roflcoopter** [comment]
This is something that was possible before but I chose to remove it when I released v2. 

The reasoning was that it was a source of confusion for some users and also the code got pretty messy.

You can make use of [YAML Anchors](https://medium.com/@kinghuang/docker-compose-anchors-aliases-extensions-a1e4105d70bd) to accomplish something similar
